nengo_viz/static/viz_slider.js

```diff
@@ -35,10 +40,12 @@ VIZ.Slider = function(parent, args) {
         slider.value = args.start_value[i];
 
         /** Show the slider Value */
-        var valueDisplay = document.createElement('p');
-        valueDisplay.classList.add('unselectable')
+        var valueDisplay = document.createElement('div');
+        valueDisplay.classList.add('value_display')
+        valueDisplay.id = 'value_display';
```

Comment:
Hmm...  I'm pretty sure HTML ```id``` values have to be unique.  I'm not sure what problems this might cause, but it might be related to the wrong slider being modified sometimes.